less/admin/ExtensionPage.less

```diff
@@ -0,0 +1,120 @@
+@extension-list-column-width: 410px;
+
+.ExtensionPage {
+  min-height: 100vh;
+
+  .ExtensionPage-header,
+  .ExtensionPage-permissions-header {
+    padding: 10px 0;
+    background: @control-bg;
+
+    h2 {
+      color: @muted-color;
+      margin-left: 35px;
+
+      span {
+        margin-left: 10px;
+        font-size: 13px;
+        color: @muted-color;
+        font-weight: normal;
+      }
+    }
+
+    button {
+      background: none;
+      border: none;
+
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+
+    ul {
+      list-style-type: none;
+      padding: 0;
+      margin: 0;
+
+      > li {
+        display: inline;
+        color: @muted-color;
+        margin-right: 7px;
+
+        > a {
+          color: @muted-color;
+        }
+
+        > .icon {
+          margin-right: 5px;
+        }
+      }
+    }
+
+    .ExtensionPage-headerItems {
+      padding-top: 15px;
+
+      .ExtensionInfo {
+        padding-top: 5px;
+      }
+
+      @media @phone {
+        .ExtensionPage-headerActionItems {
+          padding-top: 20px;
+        }
+      }
+    }
+
+    .Checkbox.off {
+      .Checkbox-display {
+        background: @muted-more-color;
+      }
+    }
+
+    .ExtensionInfo {
+      float: right;
+    }
+
+    .ExtensionIcon {
+      width: 25px;
+      height: 25px;
+      background: @control-bg;
+      color: @control-color;
+      border-radius: 4px;
+      display: inline-block;
+      font-size: 12.5px;
+      line-height: 25px;
+      text-align: center;
+      position: absolute;
+      margin-left: -35px;
```

Comment:
Because `.ExtensionIcon` has already been declared and can be used globally, most rules here don't need to be redeclared, the only additions/modifications seem to be with `width` `height` `position` `margin-left` `font-size`

On another note, why do we have to use an absolute position here ? In most cases we can accomplish anything without having to resort to it. (makes maintaining the code and future design changes hard compared to a more straightforward and simple approach)

Also the icon in the extension page is very small, we could probably up that to around 40px.